crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,26 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version_{}_{}",
+                qtbuild.version().major,
+                minor
+            );
+            println!("cargo:rustc-cfg={}", at_least_qt_version);
+            let variable_cargo = format!("CARGO_CFG_{}", at_least_qt_version);
+            env::set_var(variable_cargo, "true");
+        }
+
+        // We don't support Qt < 5
+        (5..qtbuild.version().major).for_each(|_: u32| {
+            let at_least_qt_major_version =
+                format!("cxxqt_at_least_qt_version_{}", qtbuild.version().major);
```

Comment:
this should not be `qtbuild.version().major` but the `u32` value in the for loop